addons/subscription/security/ir.model.access.csv

```diff
@@ -1,7 +1,7 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_subscription_document_fields,subscription.document.fields,model_subscription_document_fields,base.group_tool_manager,1,1,1,1
-access_subscription_subscription_user,subscription.subscription user,model_subscription_subscription,base.group_tool_user,1,1,1,1
-access_subscription_subscription_history_user,subscription.subscription.history user,model_subscription_subscription_history,base.group_tool_user,1,1,1,1
-access_subscription_document_user,subscription.document user,model_subscription_document,base.group_tool_user,1,1,1,1
-access_res_partner_user,res.partner.user,base.model_res_partner,base.group_tool_user,1,1,1,1
-access_ir_cron_user,ir.cron.user,base.model_ir_cron,base.group_tool_user,1,1,1,1
+access_subscription_document_fields,subscription.document.fields,model_subscription_document_fields,,1,1,1,1
+access_subscription_subscription_user,subscription.subscription user,model_subscription_subscription,,1,1,1,1
+access_subscription_subscription_history_user,subscription.subscription.history user,model_subscription_subscription_history,,1,1,1,1
+access_subscription_document_user,subscription.document user,model_subscription_document,,1,1,1,1
+access_res_partner_user,res.partner.user,base.model_res_partner,,1,1,1,1

```

Comment:
it should be read access. 
like this
access_res_partner_user,res.partner.user,base.model_res_partner,,1,0,0,0